test/TestDSP.c

```diff
@@ -102,6 +112,19 @@ void test_dsp_object(void)
     TEST_ASSERT_DOUBLE_WITHIN(0.01, -16.653, dspv5_object_helper(-5000, -5000, 22454, 23030));
 }
 
+void test_dsp_object_reflected(void)
+{
+    double ke = mlx90632_get_emissivity();
```

Comment:
This is not needed as SetUp is run before each test, so there is no need to store and reset after each test.